File: CBASIC/UtilFiles/BankingSystem.py
# Simple Banking System

def banking_system():
    balance = 1000  # Initial balance
    print("Welcome to the Banking System")

    while True:
        print(f"Current Balance: {balance}")
        print("1. Deposit")
        print("2. Withdraw")
        print("3. Exit")
        
        choice = int(input("Enter your choice: "))
        
        if choice == 1:
            amount = float(input("Enter deposit amount: "))
            if amount > 0:
                balance += amount
                print(f"Deposited: {amount}")
            else:
                print("Invalid amount.")
        
        elif choice == 2:
            amount = float(input("Enter withdrawal amount: "))
            if amount > 0 and amount <= balance:
                balance -= amount
                print(f"Withdrawn: {amount}")
            else:
                print("Invalid amount or insufficient funds.")
        
        elif choice == 3:
            print("Thank you for using the Banking System.")
            break
        
        else:
            print("Invalid choice. Please try again.")

# Run the banking system
banking_system()

File: CBASIC/UtilFiles/BankingSystem.js
// Simple Banking System

function bankingSystem() {
    let balance = 1000;  // Initial balance
    console.log("Welcome to the Banking System");

    while (true) {
        console.log(`Current Balance: ${balance}`);
        console.log("1. Deposit");
        console.log("2. Withdraw");
        console.log("3. Exit");

        let choice = parseInt(prompt("Enter your choice: "), 10);

        if (choice === 1) {
            let amount = parseFloat(prompt("Enter deposit amount: "));
            if (amount > 0) {
                balance += amount;
                console.log(`Deposited: ${amount}`);
            } else {
                console.log("Invalid amount.");
            }
        } else if (choice === 2) {
            let amount = parseFloat(prompt("Enter withdrawal amount: "));
            if (amount > 0 && amount <= balance) {
                balance -= amount;
                console.log(`Withdrawn: ${amount}`);
            } else {
                console.log("Invalid amount or insufficient funds.");
            }
        } else if (choice === 3) {
            console.log("Thank you for using the Banking System.");
            break;
        } else {
            console.log("Invalid choice. Please try again.");
        }
    }
}

// Run the banking system
bankingSystem();
